book/src/example-numbat_syntax.md

```diff
@@ -103,6 +103,7 @@ print(2 kilowarhol)              # Print the value of an expression
 print("hello world")             # Print a message
 print("value of pi = {pi}")      # String interpolation
 print("sqrt(10) = {sqrt(10)}")   # Expressions in string interpolation
+print("value of e ~= {e:.3}")    # Format specifiers
```

Comment:
Minor: maybe
```suggestion
print("value of e ≈ {e:.3}")     # Format specifiers
```

Reply:
Also, let's use `pi` instead of `e`. I think `e` can be confusing here because it's also used in scientific notation.
```suggestion
print("value of π ≈ {π:.3}")     # Format specifiers
```